WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingStreamsViewController.swift

```diff
@@ -155,6 +156,20 @@ public class NotificationSettingStreamsViewController : UITableViewController
         return sortedStreams![section]
     }
     
+    private func imageForStreamKind(streamKind: NotificationSettings.Stream.Kind) -> UIImage? {
+        let imageName : String
+        switch streamKind {
+        case .Email:
+            imageName = "notifications-email"
+        case .Timeline:
+            imageName = "notifications-bell"
+        case .Device:
+            imageName = "notifications-phone"
+        }
```

Comment:
Maybe shortcut and return nil? Just thinking that passing in a nil string to UIImage(named:) might throw an error or exception.